packages/search/lib/indexPgTable.js

```diff
@@ -0,0 +1,80 @@
+// Amount of rows, documents in a single bulk request.
+const BULK_SIZE = 10000
+
+// Interval in seconds stats about bulk progress is printed.
+const STATS_INTERVAL_SECS = 2
+
+const index = async ({ indexName, type, elastic, resource }) => {
+  const stats = { [type]: { added: 0 } }
+  const statsInterval = setInterval(() => {
+    console.log(indexName, stats)
+  }, STATS_INTERVAL_SECS * 1000)
+
+  let offset = 0
+  let rows = []
+
+  do {
+    rows = await resource.table.find(
+      {},
+      {
+        orderBy: { id: 'asc' },
+        limit: resource.bulkSize || BULK_SIZE,
+        offset
+      }
+    )
+
+    if (resource.transform) rows = rows.map(resource.transform)
+
+    await bulk({ indexName, type, elastic, rows })
+
+    stats[type].added += rows.length
+    offset += BULK_SIZE
+  } while (rows.length >= BULK_SIZE)
+
+  clearInterval(statsInterval)
+
+  console.log(indexName, Object.assign(stats, { done: true }))
+}
+
+/**
+ * Prepares ElasticSearch-compatible documents of passed <rows> and does submit
+ * documents in "bulk" to ElasticSearch.
+ *
+ * @param  {String}  indexName
+ * @param  {String}  type
+ * @param  {Object}  elastic
+ * @param  {Array}   rows
+ * @return {Promise}
+ */
+const bulk = async ({ indexName, type, elastic, rows }) => {
+  const bulk = {
```

Comment:
Renamed `bulk` in `bulk` 💅 see d68afee

Reply:
StandardJS has not yet a rule on shadowing variables but there is an (old) [issue pending](https://github.com/standard/standard/issues/711).

I believe a sane way forward could be [ESLint](https://eslint.org/), using "standard" and extending it with a specific set of rules, like [no-shadow](https://eslint.org/docs/rules/no-shadow). [republik-admin-frontend](https://github.com/orbiting/republik-admin-frontend) seems [to use ESLint](https://github.com/orbiting/republik-admin-frontend/blob/master/.eslintrc). 

It would probably take a little while to agree on a set of rules. I have no strong opinions on how to format and lint, really, but find it tempting to something enforced not onkly repo but project-wide.

And on a funny note: Using ESLint with StandardJS and additional rules does torpedoe StanardJS's promise of "No decisions to make. (...) No configuration.", basically a set of rules and not deviation from that, no discussions, no nothing.